docker-compose/high-availability/test.sh

```diff
@@ -10,85 +10,95 @@ echo "running tests"
 #build latest image
 docker build -t timescale/promscale:latest ../.. --file ../../build/Dockerfile
 
-docker-compose -p promscale_deploy_ha up -d
+docker compose -p promscale_deploy_ha up -d
```

Comment:
Not sure if this is expected, but this is how my system responds

```shell
harkishen@harkishen:~/go/src/github.com/timescale/promscale$ docker compose
docker: 'compose' is not a docker command.
```

Reply:
Update Docker.